### DISCUSSION TITLE
Can Plausible be hosted on Lambda + DynamoDB?

### DISCUSSION THREAD

**Discussion by alshdavid** [plausible/analytics#2829] [Self-Hosted Support]
I have a static website hosted on S3, which is a choice made to essentially have a free web host for basic static content and minimize the complexity in its deployment/maintenance.

I am interested in gathering analytics and I love Plausible as a platform. Unfortunately, the $6/m fee (~10 AUD) is higher than the cost of a basic VPS and in many ways defeats the purpose of a free static website.

I would love to use Plausible but it doesn't make sense for my use case.

In general, Lambda and DynamoDB offer a pay-per-usage model - considering the low traffic nature of my site, the free tiers would likely result in no costs associated if I could use Plausible in that context.

Looking over the source code, there doesn't seem to be an obvious way to adapt a self hosted Plausible instance to a serverless model.

Has anyone else done this?

Is this Plausable? (pardon the pun :laughing:)

**Comment by victorberland** [comment]
I currently host Plausible myself and have three Docker containers running as part of the stack — Postgres, Clickhouse and Plausible. The Plausible container is stateless, so that one could run in a container service, but you would need an external service for Postgres and Clickhouse. Most providers offer managed databases with Postgres, but I haven't found many that offer Clickhouse aside from Clickhouse themselves (https://clickhouse.com/pricing).
